### DISCUSSION TITLE
Location of PCB files?

### DISCUSSION THREAD

**Discussion by MrAureliusR** [frntc/SIDKick-pico#81] [General]
Hi there!

Thanks for releasing your hardware under CC-BY-SA-NC-ND (I think that's the correct version you chose). Is it because of the ND part that you don't have the PCB files available somewhere? I'd much, much prefer to support a different PCB manufacturer than PCBWay as they are not exactly the best company around... I'd much rather support a grassroots, hacker-based PCB house like DirtyPCBs, as they contribute back to the community in a huge way.

If you don't want to release the PCB files publicly, I believe there is a way to list your design on the Hacker Store on DirtyPCBs, similar to how you can do so on PCBWay. The folks who run DirtyPCBs (including Ian who also runs and owns DangerousPrototypes) are simply amazing, generous, customer-focused people who go out of their way to exceed your expectations. 

Hoping there can be some way we can choose different board houses in the future :) I also believe that OSHPark has a similar listing service. And, perhaps at some point, you might consider releasing the project under a more permissive license, and becoming Certified Open Source Hardware at OSHWA! 

Cheers and thanks so much for this awesome project, going to build at least one if I can get some boards or figure out the schematic somehow to homebrew a protoboard!

**Comment by frntc** [comment]
I get you points - I'm open to other platforms and happy to look at DirtyPCB when I find some time, but for now I'd like to stick to PCBWay. The reason for not publishing KiCAD-projects/gerbers mainly being the "NC": I have been annoyed too often in the past by (a few) individuals who ignore licenses and sell NC projects with the intention of making a profit. With PCBWay there remains some control at least.

**Reply by MrAureliusR** [reply]
Fair enough. It's 5AM here right now but I will try and remember to come back and make a few points -- not to be antagonistic or critical in any way, just that I have experience with open source and open hardware and I think it's a beautiful thing :)
I totally understand your concerns. I'd love to contribute to the project somehow if possible as well! In any case, I really want to make a couple of your boards, one for my second (currently non-working) C64, and another for my Midibox SID synth that I built years ago that also needs repair. Back then I didn't understand the difference between the two SID versions and I think I likely fried a 6581 by inserting while the board was configured for an 8580, or vice versa? I'll have to dig it out and see.
